config/development/params.yaml

```diff
@@ -15,10 +14,6 @@ footer_scripts:
   - snowplow: false
   - adroll: false
 
-datadog:
-  clientToken: pub36877d3864fab670b5ae7e1d5d26cb0
-  logging_handler: console
-
 algolia:
```

Comment:
Since the algolia config params are in `src/scripts/config/config-docs.js` we could remove them from here as well ?

Reply:
removed.